rest/attachment_test.go

```diff
@@ -575,7 +575,7 @@ func TestAttachmentsNoCrossTalk(t *testing.T) {
 	}
 
 	// attach to existing document with correct rev (should succeed)
-	response := rt.SendAdminRequestWithHeaders("PUT", "/{{.keyspace}}/doc1/attach1?rev="+doc1revId, attachmentBody, reqHeaders)
+	response := rt.SendAdminRequestWithHeaders("PUT", "/{{.keyspace}}/doc1/attach1?rev="+doc1Version.RevID, attachmentBody, reqHeaders)
```

Comment:
Just to confirm - is an update to the handling on line 583 planned to be included in a follow-up PR?